engine/src/main/java/org/terasology/logic/characters/CharacterSoundSystem.java

```diff
@@ -105,8 +107,17 @@ public void onJump(JumpEvent event, EntityRef entity, CharacterSoundComponent ch
         }
     }
 
+    /**
+     * React to any vertical collison of a character that has
+     * a component for character sounds and decide what the volume
+     * of the landing sound should be and the sounds should pe played.
+     * Then send a PlaysSoundEvent
+     * @param event the event that triggered the vertical collision
+     * @param entity the entity that collide vertical
+     * @param characterSounds the component that contains info about character sounds
```

Comment:
`@param characterSounds The component that contains information on the character sounds.` 

Can we use full sentences with punctuation and the first letter capital not just here, but everywhere (including the two cases right above this one)? It makes things much more natural to read, which is very important in documentation :)